LPD 25954

ealonso opened the pull request:
- LPD-25954 Allow specifying nested items for dropdown items
- LPD-25954 Add recursive mapper method, so it supports nested items
- LPD-25954 Baseline

ruben-pulido commented:
/cc @ruben-pulido <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 committed:
LPD-25954 Allow specifying nested items for dropdown items

LPD-25954 Baseline

# breaking

## What /modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/util/DropdownGroupItem.java

public void setDropdownItems(List<DropdownItem> dropdownItems), was deleted

## Why

This method was moved to DropdownItemBuilder

----

LPD-25954 Baseline

# breaking

## What /modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/util/DropdownItemBuilder.java

DropdownItemsStep attribute was added to several methods

## Why

Add more flexibility when using Clay Dropdown taglib

----

sandrodw3 committed:
LPD-25954 Add recursive mapper method, so it supports nested items

sandrodw3 committed:
LPD-25954 Baseline

Allow using nested items in Clay Dropdown taglib, making possible to have different sublevel menus

Add more flexibility when using Clay Dropdown taglib

----

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
